chromium_src/chrome/browser/ui/webui/settings/settings_ui.cc

```diff
@@ -20,7 +21,9 @@
 #define SiteSettingsHandler BraveSiteSettingsHandler
 #define ImportDataHandler BraveImportDataHandler
 #define SearchEnginesHandler BraveSearchEnginesHandler
+#define ClearBrowsingDataHandler BraveClearBrowsingDataHandler
```

Comment:
Should probably also #include "chrome/browser/ui/webui/settings/settings_clear_browsing_data_handler.h" so that `ClearBrowsingDataHandler` isn't redefined in there.

Reply:
Although, I guess browser/ui/webui/settings/brave_clear_browsing_data_handler.h already include it.